src/Interface.js

```diff
@@ -0,0 +1,73 @@
+const root = document.documentElement;
+        
+const gridBlocks = document.getElementById("gridPage");
+let holding = false;
+
+// 2nd test object
+const ob = document.createElement("div");
+ob.className = "gridSpanTest";
```

Comment:
What is an `ob`? If you mean a generic object, the standard name is `obj`. Otherwise, use a descriptive variable name.